doc/manual/install-judgehost.rst

```diff
@@ -50,15 +50,34 @@ home directory::
   make judgehost
   sudo make install-judgehost
 
+The judgedaemon can be run on various hardware configurations;
+- A virtual machine, typicly these have 1 or 2 cores and no hyperthreading, because the kernel will schedule its own tasks on CPU 0, we advice CPU 1,
+- A default office machine, these sometimes have hyperthreading. Verify if the machine has hyperthreading and consider turning it off and as a rule of thumb pick CPU 2 as CPU 1 could be a hyperthreading core, be on the same die as CPU 0 and therefore share memory with that CPU. If more cores available as a rule of thumb moving to the highest CPU should be considered.
+- Multiple on a single high-grade server with multiple CPUs or a CPU with multiple cores. Check for hyperthreading and if possible run the judgedaemon on another CPU package or CPU die than the CPU 0. See the section :ref:`multiple-judgedaemons` for running multiple
```

Comment:
Fixes: "run the judgedaemon*s* on separate CPU packages/dies and different from CPU 0".
I think in this case you mostly want the different judgedaemons to not interfere with each other. The load of the OS itself is less relevant.

Reply:
The section was meant with only 1 judgedaemon, but I think your interpretation works better.